src/io/flutter/vmService/HeapMonitor.java

```diff
@@ -52,11 +49,13 @@ String getAsString(String memberName) {
     }
 
     JsonObject getAsJsonObject(String memberName) {
-      return json.get(memberName).getAsJsonObject();
+      final JsonElement element = json.get(memberName);
```

Comment:
From some slight perusing of the source code, we may also fall down when trying to read `"_currentRSS"` out of the `VM` object? I assume none of this info is populated for flutter web.

Reply:
Good catch!